templates/epic.xml.jinja2

```diff
@@ -154,6 +154,7 @@
     ## Central beam pipe
   </documentation>
   <include ref="${DETECTOR_PATH}/compact/central_beampipe.xml"/>
+  <include ref="${DETECTOR_PATH}/compact/forward_electron_beampipe.xml"/>
```

Comment:
This should not go into the jinja template directly. It should go into the far_forward stack.

Reply:
Addressed in the new commit